apps/frontend/src/app/3_organisms/ZeroLocForm/components/AdjustCreditLine.tsx

```diff
@@ -351,17 +355,26 @@ export const AdjustCreditLine: FC<AdjustCreditLineProps> = ({
     collateralAmount,
     debtAmount,
     debtToken,
+    maxOriginationFeeRate,
     isIncreasingCollateral,
     isIncreasingDebt,
     onSubmit,
   ]);
 
+  useEffect(() => {
```

Comment:
This is not the correct way how to do it - since you initialize the value to `0` and then change it in `useEffect`, there will be a re-render and it can cause some bugs/flickering. Just assign the value in the initializer (line `57`).

The same applies to `OpenCreditLine`.